Product/Production/Common/Properties/src/main/resources/gateway.properties

```diff
@@ -118,9 +118,8 @@ CoreX12GenericBatchTimeStampTimeToLive=300
 CoreX12GenericBatchFutureTimeToLive=60
 
 # Boolean for adding the default values SAML Before and After Conditions. Default property value is true.
+# Recommend only disabling for testing purposes, SAML conditions can cause validation errors if set incorrectly
 enableConditionsDefaultValue=true
```

Comment:
need to remove the property enableAuthDecEvidenceConditionsDefaultValue from NhincConstants.java